config/i18n-tasks.yml

```diff
@@ -126,6 +126,7 @@ ignore_unused:
   - 'user_mailer.email_confirmation_instructions.subject'
   - 'valid_email.validations.email.invalid'
   - 'friendly_errors.*'
+  - 'doc_auth.*'
```

Comment:
I don't love this, but it's the easiest short-term option.

Longer-term:

- Move all string keys in `i18n-strings.js.erb` to a `.yml` file, change `i18n-tasks.yml` to `i18n-tasks.yml.erb`, and merge the strings into this list automatically
- [Custom scanner](https://github.com/glebm/i18n-tasks/wiki/A-custom-scanner-example) implementation to check for a pattern we're expecting in the `i18n-strings.js.erb`

Might tie into some other work I'd like to do to try to eliminate `i18n-strings.js.erb` in favor of a bundle / middleware task to create locale-specific files.

Reply:
It occurred to me later that this is the same problem as [described as "Dynamic keys" documentation for Ruby usage](https://github.com/glebm/i18n-tasks#dynamic-keys), where `i18n-tasks unused` _is_ capable of finding the strings in JavaScript, but since they were dynamic as e.g. ``t(`doc_auth.headings.${side}`)``, they were being flagged as unused. As suggested in the documentation, I chose to update this in 2a0ac76 to use the [usage hints](https://github.com/glebm/i18n-tasks#fine-tuning).